MdeModulePkg/Core/Dxe/Event/Tpl.c

```diff
@@ -9,6 +9,16 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include "DxeMain.h"
 #include "Event.h"
 
+//
+// Bit mask of TPLs that were interrupted (typically during RestoreTPL's
+// event dispatching, though there are reports that the Windows boot loader
+// executes stray STIs at TPL_HIGH_LEVEL).  CoreRaiseTpl() sets the

```

Comment:
Could we please avoid X86-specific lingo here? This code is shared by all architectures, and terms like `STI` and `IRET` may not be familiar to everyone.